weblate/vcs/git.py

```diff
@@ -654,18 +690,29 @@ def create_pull_request(self, origin_branch, fork_remote, fork_branch):
             head = fork_branch
         else:
             head = "{0}:{1}".format(fork_remote, fork_branch)
-        self.execute(
-            [
-                "pull-request",
-                "--force",
-                "--head",
-                head,
-                "--base",
-                origin_branch,
-                "--message",
-                self.get_merge_message(),
-            ]
+        pr_url = "{}/pulls".format(self.api_url())
+        r = requests.post(
+            pr_url,
+            headers={
+                "Accept": "application/vnd.github.v3+json",
+                "Authorization": "token {}".format(settings.GITHUB_TOKEN),
+            },
+            json={
+                "head": head,
+                "base": origin_branch,
+                "title": self.get_merge_message(),
+            },
         )
+        response = r.json()
+        if "url" not in response:
+            for error in response["errors"]:
+                if "message" in error:
+                    report_error(cause=error["message"])
+                else:
+                    report_error(
+                        cause="{0}: {1}".format(error["resource"], response["message"]),
+                        level="error",
+                    )
```

Comment:
I'd probably do it just:

```suggestion
            report_error(cause="Pull request failed", extra_data={"errors": response["errors"]})
```

Reply:
:+1: